knowledge-base/pdfprocessing-creating-tablecells-with-gotoaction-and-uriaction.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Creating TableCells with GoToAction and UriAction
+description: This article demonstrates how to create table cells with GoToAction and UriAction in PDF document with Telerik Document Processing.
+type: how-to 
+page_title: Creating TableCells with GoToAction and UriAction
+slug: kb-create-table-cells
+position: 0
+tags: pdf, cells, actions
+res_type: kb
+---
+
+|Product Version|Product|Author|
+|----|----|----|
+|N/A|RadPdfProcessing|Maria Terzieva|
+
+## Description
+
+With the PDF format, each content element has a specific position, and in some cases, each character is positioned separately. PDF standard does not have any information about tables as well, once exported the tables in the document are represented by lines and text fragments. 
+For this reason we use Action which defines the behaviour for an annotation. And annotation element associates an object with a location on a RadFixedPage.
```

Comment:
I am still not sure I can understand why actions are needed and what this example shows from the description.